### PR TITLE
[10-09]Update Simplified Chinese translation

### PR DESCRIPTION
**Pull request by njlyf2011**

### CODE REVIEW

File: dll/win32/shell32/lang/zh-CN.rc [hunk 1]
```diff
@@ -650,74 +650,75 @@ BEGIN
...
     AUTOCHECKBOX "Confirm &open after download", IDC_EDITTYPE_CONFIRM_OPEN, 5, 115, 150, 14
-    AUTOCHECKBOX "Al&ways show extension", IDC_EDITTYPE_SHOW_EXT, 5, 130, 150, 14
-    AUTOCHECKBOX "&Browse in same window", IDC_EDITTYPE_SAME_WINDOW, 5, 145, 150, 14
-    DEFPUSHBUTTON "OK", IDOK, 95, 170, 60, 14
-    PUSHBUTTON "Cancel", IDCANCEL, 160, 170, 60, 14
+    AUTOCHECKBOX "总是显示插件(&w)", IDC_EDITTYPE_SHOW_EXT, 5, 130, 150, 
+14
+    AUTOCHECKBOX "在相同窗口中浏览(&B)", IDC_EDITTYPE_SAME_WINDOW, 5, 145, 150, 14
+    DEFPUSHBUTTON "确定", IDOK, 95, 170, 60, 14
+    PUSHBUTTON "取消, IDCANCEL, 160, 170, 60, 14
```
Comment (Fraizeraust) [inline dll/win32/shell32/lang/zh-CN.rc]: The string is missing the last double quote.

File: dll/win32/shell32/lang/zh-CN.rc [hunk 2]
```diff
@@ -650,74 +650,75 @@ BEGIN
...
     COMBOBOX IDC_FOLDERCUST_COMBOBOX, 15, 35, 210, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    AUTOCHECKBOX "Also apply this template to all &subfolders", IDC_FOLDERCUST_CHECKBOX, 15, 50, 210, 15
-    GROUPBOX "Folder pictures", IDC_STATIC, 5, 75, 230, 90, WS_TABSTOP
-    LTEXT "For Thumbnails view, you can put a picture on this folder to remind you of the contents.", IDC_STATIC, 15, 87, 115, 33
-    PUSHBUTTON "Choose &Picture...", IDC_FOLDERCUST_CHOOSE_PIC, 15, 125, 115, 15
-    PUSHBUTTON "&Restore Default", IDC_FOLDERCUST_RESTORE_DEFAULTS, 15, 144, 115, 15
-    LTEXT "Preview:", IDC_STATIC, 139, 85, 81, 11
+    AUTOCHECKBOX "将此模板应用到所有子文件夹(&s)", IDC_FOLDERCUST_CHECKBOX, 15, 50, 210, 15
+    GROUPBOX "文件夹图片", IDC_STATIC, 5, 75, 230, 90, WS_TABSTOP
+    LTEXT "对于缩略图视图，您可以在此文件夹上放置能够提醒您的图片。, IDC_STATIC, 15, 87, 115, 33
```
Comment (Fraizeraust) [inline dll/win32/shell32/lang/zh-CN.rc]: Ditto.

**Comment by njlyf2011** [comment]
Sorry, I will fix it later.

**Comment by njlyf2011** [comment]
Fixed.

**Comment by Andy312432** [comment]
只能有一個commit，我就是因為有3個commit被退件。(closed pull request)

You can only have a commit.

**Comment by gigaherz** [comment]
Err, we recommend that pull requests be squashed into one commit, but having multiple commits is fine. No need to close the PR!

Also, you can do a squash locally and force-push to the same branch, so no need to close either way!

**Comment by njlyf2011** [comment]
Closed.I will re-create it.

**Pull request closed without merging**